README.md

```diff
@@ -394,6 +394,15 @@ By default, the exported JSON keys mirror the class names. If you want to cameli
 import { className } from 'file.css';
 ```
 
+#### Possible options
+
+|Option|Behaviour|
```

Comment:
Move the table directly under the `camelCase` header please

`Behaviour` => `Description` for consistency